### PR TITLE
[hjson/en] add HJSON

### PR DESCRIPTION
**Pull request by MrTeferi**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]` (example `[python/fr-fr]` or `[java/en]`)
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: hjson.html.markdown [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+language: hjson
```
Comment (mribeirodantas) [inline hjson.html.markdown:2]:
```suggestion
language: Hjson
```

File: hjson.html.markdown [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+filename: learnhjson.hjson
+contributors:
+  - ["MrTeferi", "https://github.com/MrTeferi"]
+lang: en
+---
+
+HJSON is an attempt to make [JSON](https://learnxinyminutes.com/docs/json/) more human readable.
```
Comment (mribeirodantas) [inline hjson.html.markdown:9]:
```suggestion
Hjson is an attempt to make [JSON](https://learnxinyminutes.com/docs/json/) more human readable.
```

File: hjson.html.markdown [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+HJSON is a syntax extension to JSON.
```
Comment (mribeirodantas) [inline hjson.html.markdown:11]:
```suggestion
Hjson is a syntax extension to JSON. 
```

Comment (mribeirodantas) [inline hjson.html.markdown:11]:
```suggestion
Hjson is a syntax extension to JSON. 
```

File: hjson.html.markdown [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+        ]
+        details: ["this", "is", "fine", "too"]
+    }
+    
+    # Multiline quotes with proper whitespace handling are supported!
+    diary:
+        '''
+        I wish JSON was more human readable.
+        If only there was a JSON for my needs!
+        Oh wait.. there is! It's called HJSON.
```
Comment (mribeirodantas) [inline hjson.html.markdown:55]:
```suggestion
        Oh wait.. there is! It's called Hjson.
```

File: hjson.html.markdown [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+    
+    # Strings do not require quotes!
+    # Just keep it to a single line
+    human: readable
+    quotes: "are fine too"
+    
+    # Notice that commas are also not required!
+    # If using commas, strings DO require quotes!
+    object: {
+        name: hjson
```
Comment (mribeirodantas) [inline hjson.html.markdown:36]:
```suggestion
        name: Hjson
```

File: hjson.html.markdown [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+    slash: This will not have a new line\n
+    slash-quoted: "This will definitely have a new line\n"
+
+    # Make sure to use quotes when mixing whitespace with important punctuation
+    example1: "If, you're, going, to, comma in a string, use, quotes!"
+    example2: "Also if you want to use {} or [] or any JSON relevant punctuation!"
+    example3: [because, this, is, totally, BROKEN!]
+    example4: this is technically OK though: {}[],:
+    
+    # Enjoy working with HJSON!
```
Comment (mribeirodantas) [inline hjson.html.markdown:68]:
```suggestion
    # Enjoy working with Hjson!
```

File: hjson.html.markdown [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+        }
+        partial-support: ["C", "Kotlin", "Ruby", "Rust"]
+    }
+    
+}
+```
+
+## Further Reading
+
+* [HJSON.github.io](https://hjson.github.io/) Main HJSON site including editor support, how-to, etc.
```
Comment (mribeirodantas) [inline hjson.html.markdown:93]:
```suggestion
* [Hjson.github.io](https://hjson.github.io/) Main Hjson site including editor support, how-to, etc.
```

File: hjson.html.markdown [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+* [HJSON Packages](https://github.com/hjson/) Various HJSON packages for different applications.
```
Comment (mribeirodantas) [inline hjson.html.markdown:94]:
```suggestion
* [Hjson Packages](https://github.com/hjson/) Various Hjson packages for different applications.
```

File: hjson.html.markdown [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+
+    # Make sure to use quotes when mixing whitespace with important punctuation
+    example1: "If, you're, going, to, comma in a string, use, quotes!"
+    example2: "Also if you want to use {} or [] or any JSON relevant punctuation!"
+    example3: [because, this, is, totally, BROKEN!]
+    example4: this is technically OK though: {}[],:
+    
+    # Enjoy working with HJSON!
+    party-time: {
+        hjson-lovers: [
```
Comment (mribeirodantas) [inline hjson.html.markdown:70]:
```suggestion
        Hjson-lovers: [
```

File: hjson.html.markdown [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+            me
+            my mom
+            "my dad"
+        ]
+        hjson-power-level: 9000
```
Comment (mribeirodantas) [inline hjson.html.markdown:75]:
```suggestion
        Hjson-power-level: 9000
```

**Review by mribeirodantas — CHANGES_REQUESTED** [review]
According to the official documentation, the proper capitalization is Hjson. Could you please accept such change request in order to get this PR merged?

File: hjson.html.markdown (hunk 3 above)
Comment (MrTeferi) [reply hjson.html.markdown]: Alrighty I think it should be good to go now!

**Pull request merged**